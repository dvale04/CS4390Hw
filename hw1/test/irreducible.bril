@main {
  jmp .A;

.A:
  br true .B .exit;

.B:
  jmp .C;

.C:
  br true .A .exit;

.exit:
  ret;
}
