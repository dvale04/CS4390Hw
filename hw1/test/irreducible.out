digraph main {
 b0;
 A;
 B;
 C;
 exit;
 b0 -> A;
 A -> B;
 A -> exit;
 B -> C;
 C -> A;
 C -> exit;
}
